modules/skeletonViewer/src/main.cpp

```diff
@@ -711,7 +711,19 @@ class Viewer : public RFModule, public skeletonViewer_IDL
             vtk_floor->SetOrigin(-5.0,-5.0,0.0);
             vtk_floor->SetPoint1(5.0,-5.0,0.0);
             vtk_floor->SetPoint2(-5.0,5.0,0.0);
-            vtk_floor->SetResolution(10,10);
+
+            std::array<int,2> grid_resolution = {10, 10};
+            if (rf.check("grid-resolution"))
+            {
+                if (const Bottle *ptr=rf.find("grid-resolution").asList())
+                {
+                    size_t len=std::min(grid_resolution.size(),ptr->size());
+                    for (size_t i=0; i<len; i++)
+                        grid_resolution[i]=ptr->get(i).asInt32();
+                }
+            }
+            vtk_floor->SetResolution(grid_resolution[0], grid_resolution[1]);
```

Comment:
Missing documentation in `skeletonViewer.xml`.

Reply:
Added documentation